views/DataEntryView.html

```diff
@@ -0,0 +1,25 @@
+
+<table class="pure-table" style="margin-left: -10px" >
+    <tr>
+        <td>
+        File Name    
+        </td> 
+        <td>
+        <form class="pure-form"><input type="text"></form>  
+        </td> 
+    </tr>
+    <tr>
+        <td>
+        Comment    
+        </td> 
+        <td>
+        <form class="pure-form"><input type="text"></form>     
+        </td> 
+        
+    </tr>
+</table>
+<div class="wrapper">
+    <button id="sendall" class="pure-button button-secondary" style="margin-top: 50px" >Save Comment</button>
```

Comment:
usually you want to avoid inline html styling as much as possible. Its fine this time but if it gets more complicated create a seperate stylesheet (css file) for the view and put it on there